pr_agent/git_providers/github_provider.py

```diff
@@ -39,6 +39,10 @@ def get_diff_files(self) -> list[FilePatchInfo]:
             diff_files.append(FilePatchInfo(original_file_content_str, new_file_content_str, file.patch, file.filename))
         return diff_files
 
+    def publish_description(self, pr_title: str, pr_body: str):
```

Comment:
Consider adding a check to ensure that the 'pr_title' and 'pr_body' are not empty before publishing the PR description. This will prevent empty PR descriptions from being published. [medium]